src/main/scala/com/github/unchama/buildassist/menu/MineStackMassCraftMenu.scala

```diff
@@ -264,9 +264,9 @@ object MineStackMassCraftMenu {
         ),
         ChestSlotRef(1, 1) -> MassCraftRecipeBlock(
```

Comment:
あ、それと `ChestSlotRef(1, 1)` はレシピブロックの開始が2行目の2列目(上から二つ目、左から二つ目; 0-indexedです)であることを指定しているのですが、 `oneToThousand` にしてボタンが4個生えるようにするなら `ChestSlotRef(1, 0)` からの開始で良さそう

```suggestion
        ChestSlotRef(1, 0) -> MassCraftRecipeBlock(
```

Reply:
わかりました、
ありがとうございます。早速修正します